apps/wallet/src/ui/styles/global.scss

```diff
@@ -65,3 +67,7 @@ input:focus,
 textarea {
     outline: none !important;
 }
+
+textarea {
```

Comment:
This rule can be merged into the one above.

Reply:
It has input:focus that indicated a different use case that's why I used a new one

Reply:
Ah didn't notice that!